### PR TITLE
Theme designer: design & layout changes

### PR DESCRIPTION
**Pull request by emmayjiang**
Making layout fixes to the theme designer and fixing the dark mode option.

image
image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/fed73f38e0f273354d66e318ad5a0513a1c3a71d'>fed73f38e0f273354d66e318ad5a0513a1c3a71d</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-components/theme-designer/src/components/Content/Content.tsx [hunk 1]
```diff
@@ -45,11 +45,25 @@ export const brandWeb: BrandVariants = {
 
 export const Content: React.FC<ContentProps> = props => {
   const styles = useStyles();
+  const [isDark, setIsDark] = React.useState<boolean>(false);
+
+  const brand = brandWeb;
```
Comment (micahgodbolt) [inline packages/react-components/theme-designer/src/components/Content/Content.tsx:50]: this assignment isn't necessary anymore

File: packages/react-components/theme-designer/src/components/TokenBoxes/TokenBoxes.tsx [hunk 2]
```diff
@@ -72,47 +76,39 @@ export const TokenBoxes: React.FC<TokenBoxesProps> = props => {
...
+          contentAfter={<Button aria-label="Search" appearance="transparent" icon={<SearchRegular />} size="small" />}
           appearance="outline"
           id={useId('input-outline')}
-          onChange={React.useCallback(
-            (ev, data) => {
-              setFilter(ev.target.value);
-            },
-            [setFilter],
-          )}
+          onChange={React.useCallback((ev, data) => setFilter(ev.target.value), [setFilter])}
```
Comment (micahgodbolt) [inline packages/react-components/theme-designer/src/components/TokenBoxes/TokenBoxes.tsx:88]: move this callback up out of the return to its own const

**Review by micahgodbolt — APPROVED** [review]
A couple small suggestions, but otherwise looks great!

**Pull request merged**